mozilla/foundation.mozilla.org
# updated copy for press and why page

Closes #7749 


Updated the copy for the press and why page. If there is any changes you would like me to make please let me know and I can get them updated asap! 


### Links to test pages:
**updated why page:** https://foundation-s-update-cop-7blkfh.herokuapp.com/en/privacynotincluded/about/why/
**updated press page:**https://foundation-s-update-cop-7blkfh.herokuapp.com/en/privacynotincluded/about/press/

### Steps to test:

1. Please visit the pages above and verify the new text matches the copy
2. If everything is looking as expected, testing is complete!
